vexim/adminuserchangesubmit.php

```diff
@@ -30,10 +30,15 @@
   }
   if (isset($_POST['on_forward'])) {
     $_POST['on_forward'] = 1;
-    if(!filter_var($_POST['forward'], FILTER_VALIDATE_EMAIL)) {
-      header ("Location: adminuser.php?invalidforward=".htmlentities($_POST['forward']));
-      die;
+    $forwardto=explode(",",$_POST['forward']);
```

Comment:
All of this assumes `$_POST['forward']` is not empty. What if it is?

Also, `$_POST['forwardmenu']` is not even checked for correctness later on, and it overwrites whatever was specified in `$_POST['forward']`. This doesn't seem very nice at all. What would you think about adding some more JS which would overwrite the contents of the `forward` field in the front-end when an address is selected in the drop-down (onchange), and then simply ignoring the contents of `$_POST['forwardmenu']` in the backend altogether? In that case, this assumption above that `$_POST['forward']` must not be empty would be correct.